src/syscall_handler_process.c

```diff
@@ -10,7 +10,7 @@ See the file LICENSE for details.
 #include "memorylayout.h" // PROCESS_ENTRY_POINT
 #include "permissions_capabilities.h"
 
-#define PROCESS_COPY_CHUNK 3500
+#define PROCESS_COPY_CHUNK 2048 // Half a page
```

Comment:
Good thought; I didn't realize PAGE_SIZE was in the namespace of the handler. Updating now